### ISSUE TITLE
Invalid value encountered during convolution 

### ISSUE THREAD

**Issue by ShaneMPutnam** [Dewberry/pfra-hydromet#22]
The following warning is produced during the first convolution step in EventsTable.ipynb: 
![image](https://user-images.githubusercontent.com/13066481/60605064-d66a9b00-9d86-11e9-8484-d3738ef2e02d.png)

This warning seems to only occur for the 6 hour duration events and only for some project areas. It does not appear to affect the results but should be investigated to identify the cause in case there is some underlying stability issue.

**Comment by slawler** [comment]
I wonder if the `per_dif_total` is returning an nan value, and that is causing this warning. The numpy [documentation ](
https://github.com/numpy/numpy/blob/9b84c1174125cb32a6be1bb6151782f8b2beda55/doc/neps/nep-0012-missing-data.rst#user-content-differences-with-numpyma) mentions this in a similar context.  

You might also try changing the settings on numpy [seterr ](https://docs.scipy.org/doc/numpy/reference/generated/numpy.seterr.html) to help isolate the issue.

**Comment by ShaneMPutnam** [comment]
Thanks! I believe I isolated the issue and am doing final testing to make sure. The warning is triggered when the total_sum or the average of the two events at each time step is equal to zero since we end up dividing a number by zero during the perc_dif calculations. As a workaround, I added an if/else statement to check if the average of the two events at each time step is equal to zero, if it is then the test statistic is set to 1 since the events are the same, else the test statistic is calculated as normal. This issue only arose when running the calculations in California since there are some events with no excess rainfall at the 6 hour duration.
